.gitignore

```diff
@@ -17,7 +17,6 @@ spec/**/*.js
 npm-debug.log
 index.html
 yarn-error.log
-yarn.lock
```

Comment:
why are you removing `yarn.lock` from .gitignore?

Reply:
it was [duplicated](https://github.com/superdesk/superdesk-client-core/pull/3459/files/012819b2af7004afc3c51ffcbe461e6d75d50e63#diff-a084b794bc0759e7a6b77810e01874f2R3), probably on a merge commit